internal/services/mssql/helper/sql_extended_auditing.go

```diff
@@ -46,21 +49,31 @@ func ExtendedAuditingSchema() *pluginsdk.Schema {
 					Optional: true,
 					Default:  true,
 				},
+				"state": {
```

Comment:
what does stat=true mean? enabled? if that is the case maybe this would be better as
```suggestion
				"enabled": {
```

Reply:
Yeah that would have been better. In the end I decided that it's not worth adding the flag though, it would increase complexity without any great benefits, so I removed.